Dockerfile

```diff
@@ -1,10 +1,9 @@
-FROM ubuntu:14.04
-MAINTAINER Wink
+FROM openjdk:8
 
-RUN apt-get update; apt-get install -y curl default-jdk
+RUN apt-get update; apt-get upgrade
 
-RUN curl -s http://d3kbcqa49mib13.cloudfront.net/spark-1.6.0-bin-hadoop2.6.tgz | tar -xz -C /usr/local/
-RUN cd /usr/local && ln -s spark-1.6.0-bin-hadoop2.6 spark
+RUN curl -s http://d3kbcqa49mib13.cloudfront.net/spark-2.1.0-bin-hadoop2.6.tgz | tar -xz -C /usr/local/
+RUN cd /usr/local && ln -s spark-2.1.0-bin-hadoop2.6 spark
```

Comment:
@kalmanchapman I'd ideally like to find a way to maintain tags or branches or similar for the different versions of spark we are running. We might have some jobs that depend on Spark 1.6 and others that need Spark 2.0. If we need to fix a problem, add a feature etc, we will need to update both clusters.